torchqdynamics/utils/utils.py

```diff
@@ -22,95 +22,106 @@
 ]
 
 
-def is_ket(x: Tensor) -> Tensor:
-    """Check if a tensor is in state vector format."""
+def is_ket(x: Tensor) -> bool:
+    """Check if a tensor is in state vector format.
+
+    Args:
+        x ( Tensor (..., d, n) ): State vector, linear map or density matrix.
+
+    Returns:
+        `True` if the last dimension `n` is `1`, `False` otherwise.
+    """
     return x.size(-1) == 1
 
 
 def ket_to_bra(x: Tensor) -> Tensor:
-    """Linear map (bra) representation of a state vector (ket).
+    r"""Linear map $\bra\psi$ representation of a state vector $\ket\psi$.
```

Comment:
```suggestion
    r"""Returns the adjoint state vector $\bra\psi$ representation of a state vector $\ket\psi$.
```

The linear map terminology is not well-known. I would call them either "bras" or "Adjoint state vectors".